apisix/plugins/cors.lua

```diff
@@ -177,13 +208,24 @@ local function set_cors_headers(conf, ctx)
     end
 end
 
-local function process_with_allow_origins(conf, ctx, req_origin)
-    local allow_origins = conf.allow_origins
+local function process_with_allow_origins(allow_origins_conf, ctx, req_origin,
+                                          cache_key, cache_version)
+    local allow_origins = allow_origins_conf
```

Comment:
We don't need to declare `allow_origins_conf` as `local allow_origins` here, just change the `allow_origins_conf` -> `allow_origins` in parameter list is ok?

Reply:
yep, changed that